bundle/cert-manager-operator/0.15.0-patch.1/cert-manager-operator.v0.15.0-patch.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,89 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    description: "x509 certificate management for Kubernetes"
+    createdAt: "2020-05-04T12:00:00Z"
+    support: "cert-manager team"
+    alm-examples: ' [{"apiVersion":"operator.cert-manager.io/v1alpha1","kind":"CertManager","metadata":{"name":"cert-manager","namespace":"cert-manager"},"spec":{}}]'
+    capabilities: Full Lifecycle
+    categories: Security
+    certified: 'true'
+    containerImage: registry.connect.redhat.com/jetstack/cert-manager-operator:v0.15.0-ubi.1
+    repository: https://github.com/jetstack/cert-manager
+  name: cert-manager-operator.v0.15.0-patch.1
```

Comment:
I don't think this is correct? 🤔

Reply:
this is the patch version which I never pushed to GitHub however it is pushed to RedHat connect so we should keep a record of it.